frontend/src/App.js

```diff
@@ -7,28 +7,45 @@ import ProgressBarForm from "./Components/ProgressBarForm";
 import SecondProgressBarForm from "./Components/SecondProgressBarForm";
 import ThirdProgressBarForm from "./Components/ThirdProgressBarForm";
 import { AuthProvider } from "./Context/AuthContext";
-import MenteeRole from "./Components/MenteeRole";
+// import MenteeRole from "./Components/MenteeRole";
+// import MentorRole from "./Components/MentorRole";
+import RoleSelection from "./Components/RoleSelection";
 
 const App = () => {
   const [alertMsg, setAlertMsg] = useState("");
 
   return (
     <AuthProvider>
-        <Routes>
-          <Route
-            path="/signup"
-            element={<SignupModal setAlertMsg={setAlertMsg} />}
-          />
-          <Route
-            path="/login"
-            element={
-              <LoginModal alertMsg={alertMsg} setAlertMsg={setAlertMsg} />
-            }
-          />
-          <Route path="/setup-profile-1" element={<ProgressBarForm />} />
-          <Route path="/setup-profile-2" element={<SecondProgressBarForm />} />
-          <Route path="/setup-profile-3" element={<ThirdProgressBarForm />} />
-          <Route path="/mentee-flow-1" element={<MenteeRole />} />
+      <Routes>
+        <Route
+          path="/signup"
+          element={<SignupModal setAlertMsg={setAlertMsg} />}
+        />
+        <Route
+          path="/login"
+          element={<LoginModal alertMsg={alertMsg} setAlertMsg={setAlertMsg} />}
+        />
+        <Route path="/setup-profile-1" element={<ProgressBarForm />} />
+        <Route path="/setup-profile-2" element={<SecondProgressBarForm />} />
+        <Route path="/setup-profile-3" element={<ThirdProgressBarForm />} />
+        <Route
+          path="/mentee-flow-1"
+          element={
+            <RoleSelection
+              lala="mentor"
+              question="What role are you interested in pursuing?"
+            />
+          }
+        />
+        <Route
+          path="/mentor-flow-1"
+          element={
+            <RoleSelection
+              lala="mentee"
+              question="What is your current role?"
+            />
+          }
```

Comment:
I think you have mixed mentor and mentee - just fix that and you should be good

Reply:
This is prop is for the replacement word on the on what that person is looking for, so ill work on my naming conventions for more clarity. Thanks Yashi!